src/components/InputTextbox/index.jsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+
+export default function index(props) {
+  const [email, setEmail] = useState('');
+
+  const handleChange = e => {
+    setEmail(e.target.value);
+  };
+
```

Comment:
The component must be independent, ephemeral, not rely on external data, or make any assumptions about what the data will be. The email is an example, not the problem. This can and will be used for other elements of the site.

Reply:
implemented props as requested.